datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -705,7 +712,8 @@ def _submit_gauges_from_summary(self, metric_name, metric, scraper_config, hostn
             custom_hostname = self._get_hostname(hostname, sample, scraper_config)
             if sample[self.SAMPLE_NAME].endswith("_sum"):
                 tags = self._metric_tags(metric_name, val, sample, scraper_config, hostname=custom_hostname)
-                self.gauge(
+                self._submit_distribution_count(
```

Comment:
nit: now that we submit sums as well we might want to rename this internal function